index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="bar-container">
+        <nav>
+           <img src="/Images/Light.png"/>
+           <img src="/Images/app_bar.png"/>
+        </nav>
+   </header>
+   <div class="container">
+       <section class="head">
+           <h2> 
+               Hey there. I’m Santiago. 
+               <div class="title"> I’m a software developer </div>
+           </h2>
```

Comment:
[OPTIONAL]

- As this will be the main heading of your website, I will recommend that you use an `h1` tag instead of` h2 `tag so that screen readers and SEO will will be able to understand and index your page properly, hence improving your website accessibility